core/src/main/java/io/aiven/klaw/controller/CacheController.java

```diff
@@ -0,0 +1,42 @@
+package io.aiven.klaw.controller;
+
+import io.aiven.klaw.config.ManageDatabase;
+import io.aiven.klaw.dao.Env;
+import io.aiven.klaw.model.ApiResponse;
+import jakarta.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/cache")
+@Slf4j
+public class CacheController {
+
+  @Autowired private ManageDatabase manageDatabase;
+
+  @PostMapping(
+      value = "/environment/tenant/{tenantId}/id/{id}",
```

Comment:
if we want to add more(teams, clusters, users .. ) in future, we would end up with more end points.
how about tenant/{tenantid}/idtype/{idType}/id/{id}

where idType could be environment/cluster/team/user etc

Reply:
Yes we definitely do, at the moment the full url is for environments
/cache/environment/tenant/{tenantId}/id/{id}

I have also started working on teams which use

/cache/team/tenant/{tenantId}/id/{id}
clusters would be

/cache/cluster/tenant/{tenantId}/id/{id}
users
/cache/user/tenant/{tenantId}/id/{id}